### PR TITLE
Add -dev suffix to Spec Version after every release

### PR DESCRIPTION
**Pull request by eksperimental**
It's misleading to have a version in master that doesn't reflect the actual released version

### CODE REVIEW

File: spec.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 title: CommonMark Spec
 author: John MacFarlane
-version: 0.20
-date: 2015-06-08
+version: 0.21-dev
+date:
```
Comment (GrahamCampbell) [inline spec.txt]: surely the shouldn't be trailing whitespace following the colon?

Comment (eksperimental) [inline spec.txt]: sorry, there is one.
I have just removed the date

Comment (eksperimental) [inline spec.txt]: or do you mean that there should not be any space after the colon?

Comment (GrahamCampbell) [inline spec.txt]: I'm saying it should be removed, yes.

Comment (eksperimental) [inline spec.txt]: yes, I can see why.
done

**Comment by jgm** [comment]
Sounds reasonable.  The hard part will be remembering to do this after each release.

**Pull request merged**